extern/sector-storage/storiface/worker.go

```diff
@@ -77,33 +93,85 @@ var _ fmt.Stringer = &CallID{}
 
 var UndefCall CallID
 
+// WorkerCalls is the Worker's RPC interface that the miner calls to schedule sealing/unsealing tasks on the worker.
+// Tasks are executed asynchronously by the worker and the response is later returned to the Miner by using the Miner's `WorkerReturn` RPC interface.
+// Miner can use the `CallID` returned by the worker to track the status of the task.
 type WorkerCalls interface {
+	// AddPiece schedules a task to add a Piece to an Unsealed sector.
+	// If this is the first Piece being added to the given Sector, the worker will create an Unsealed Sector file on
+	// one of the sealing scratch spaces in the local sector store to add the given Piece.
+	// If the given Sector already contains existing Pieces, the worker will add the given Piece to the corresponding Unsealed Sector file.
+	// If the worker does NOT have the Unsealed sector file, it will fetch it from a worker that has it before adding the Piece.
+	// The call will fail of the Worker is NOT able to allocate space to store the Unsealed sector file.
```

Comment:
```suggestion
	// The call will fail if the Worker is NOT able to allocate space to store the Unsealed sector file.
```

Reply:
> The call will fail of the Worker is NOT able to allocate space to store the Unsealed sector file

Eh, that's a bug - https://github.com/filecoin-project/lotus/issues/4069 (probably don't want to document it)

Reply:
(applies below as well)

Reply:
Ah, I guess this is correct, we're just not handling this error in the main scheduler correctly